.github/workflows/cibuild.sh

```diff
@@ -5,11 +5,11 @@ COMPILER="${COMPILER:?}"
 COMPILER_VERSION="${COMPILER_VERSION:?}"
 
 if [[ "$COMPILER" == clang ]]; then
-    CC="clang"
-    CXX="clang++"
+    CC="clang-$COMPILER_VERSION"
+    CXX="clang++-$COMPILER_VERSION"
 elif [[ "$COMPILER" == gcc ]]; then
-    CC="gcc"
-    CXX="g++"
+    CC="gcc-$COMPILER_VERSION"
+    CXX="g++-$COMPILER_VERSION"
 fi
```

Comment:
This (gcc-<version> is Ubuntu specific.

Please, use $COMPILER_VERSION only if defined. The goal is to keep cibuild.sh portable -- maybe one day we will use it also for other distros.

Reply:
Understood, fixed.